angularjs-portal-frame/src/main/webapp/portal/main/partials/header.html

```diff
@@ -24,13 +24,16 @@
              </div>
              <ul class="nav navbar-nav hidden-xs bucky-nav-large">
                  <li>
-                    <notification-bell></notification-bell>
+                    &nbsp;
```

Comment:
Didn't want to break search, also wanted to leave a blank in case we ended up wanting to put something there, maybe the notification-bell ;)